Luke-Collection_2019-09-23.sql

```diff
@@ -0,0 +1,57 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.27)
+# Database: Luke-Collection
+# Generation Time: 2019-09-23 12:30:21 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table star-wars-characters
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `star-wars-characters`;
+
+CREATE TABLE `star-wars-characters` (
+  `Name` varchar(15) DEFAULT NULL,
+  `Alignment` varchar(8) DEFAULT NULL,
```

Comment:
Alignment could be ENUM if there are only 2 possible selections, Sith and Republic